spirv_reflect.c

```diff
@@ -3504,6 +3500,14 @@ static SpvReflectResult ParsePushConstantBlocks(
       return result;
     }
 
+    // Get minimum offset for whole Push Constant block
+    // It is not valid SPIR-V to have an empty Push Constant Block
+    p_push_constant->offset = UINT32_MAX;
+    for (uint32_t i = 0; i < p_push_constant->member_count; ++i) {
```

Comment:
Maybe pick another index var name? VS is error out on this since `i` is shadowing an existing variable also called `i`.

Reply:
...... I fixed this, just forgot to save and commit amend it seems